src/org/jitsi/impl/neomedia/rtp/MediaStreamTrackDesc.java

```diff
@@ -218,7 +218,14 @@ void update(
     {
         if (!simulcast)
         {
-            frameDesc.getRTPEncoding().setActive(true);
+            RTPEncodingDesc encoding = frameDesc.getRTPEncoding();
```

Comment:
While this looks sensible (with a minor modification: this should be done only if the received packet is not the start of a key frame), I'm wondering whether these checks are needed or not. I believe the webrtc team has confirmed that when a suspended stream is resumed, it must restart with a key frame.

Reply:
I was seeing issues with creating a conference, having a second user join (recvonly), and immediately having the first user adding a stream after ICE is connected. Sometimes VideoChannel.setRtpEncodingParameters (for user 1) would call bitrateController.update (for user 2) which would fail to send an FIR in SimulcastController because the MST was not active yet. This is with a non-simulcast stream. It appears there's a race condition between the setRtpEncodingParameters path and the MST becoming active.

For the minor modification is this what it would be: `if(!encoding.isActive() && pkt.getPayloadLength(true) > 0 && !frameDesc.isIndependent())`?